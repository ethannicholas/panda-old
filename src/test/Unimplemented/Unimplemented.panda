@abstract
class Super {
    @abstract
    method hello()
}

interface Interface {
    method goodbye()
}

@abstract
class Sub : Super (Interface) {
}

class Sub2 : Sub {
}