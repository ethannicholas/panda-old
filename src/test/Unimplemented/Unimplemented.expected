error: Unimplemented.panda:15:1: non-abstract class Sub2 does not implement abstract method hello()
error: Unimplemented.panda:15:1: non-abstract class Sub2 does not implement interface method goodbye()
2 errors
